module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaServicePrincipalCreatedObject.md

```diff
@@ -23,41 +35,55 @@ Get-EntraBetaServicePrincipalCreatedObject
 ```
 
 ## Description
-The Get-EntraBetaServicePrincipalCreatedObject cmdlet gets an object created by a service principal in Azure Active Directory (AD).
+
+The `Get-EntraBetaServicePrincipalCreatedObject` cmdlet gets an object created by a service principal in Microsoft Entra ID.
 
 ## Examples
 
-### Example 1: Retrieve the objects that were created by a service principal
+### Example 1: Retrieve the objects that created by a service principal
+
+```powershell
+ Connect-Entra -Scopes 'Application.Read.All'
+ $ServicePrincipal = Get-EntraBetaServicePrincipal -Filter "DisplayName eq '<service-principal-display-name>'"
+ Get-EntraBetaServicePrincipalCreatedObject -ObjectId $ServicePrincipal.ObjectId
```

Comment:
Confirm this example works. The command is prompting me to supply `ServicePrincipalId` instead. I am using the latest copy of `main` branch built locally.

Reply:
Get-EntraBetaServicePrincipalCreatedObject cmdlet is in  blocked state, so here just updated docs as per template. Example will not works as the cmdlet is blocked.